guides/network/request-and-response-filters.md

```diff
@@ -1,334 +0,0 @@
-# Request and Response filters
```

Comment:
Maybe instead of deleting the file, put here a link to the new file? This will avoid broken links in issues, etc.